build.gradle

```diff
@@ -60,7 +60,7 @@ dependencies {
 
```

Comment:
I think we should remove versions for dependencies on cradle (they are specified in the bom as I remember)

Reply:
No, we should remove all version for libraries related to th2 from bom.

Reply:
Okay, but I thought that the Cradle is more an external library than our internal library for th2. It is still up to you. But we need to keep in mind that the `common` depends on the Cradle as well if I am not mistaken